src/components/application_manager/src/resumption/resumption_data_processor.cc

```diff
@@ -788,6 +822,26 @@ void ResumptionDataProcessor::DeleteButtonsSubscriptions(
   }
 }
 
+void ResumptionDataProcessor::DeleteWindowsSubscriptions(
+    ApplicationSharedPtr application) {
+  LOG4CXX_AUTO_TRACE(logger_);
+
+  const auto window_ids = application->GetWindowIds();
+  for (const auto& window_id : window_ids) {
+    const app_mngr::HmiStatePtr hmi_state =
+        application->CurrentHmiState(window_id);
+    if (mobile_apis::WindowType::WIDGET != hmi_state->window_type()) {
+      continue;
+    }
+
+    LOG4CXX_DEBUG(logger_, "Reverting CreateWindow for: " << window_id);
+
+    auto delete_request = MessageHelper::CreateUIDeleteWindowRequestToHMI(
+        application, application_manager_, window_id);
+    ProcessHMIRequest(delete_request, false);
```

Comment:
Confusing `false`.

Reply:
@LuxoftAKutsan moved out to a separate variable. See https://github.com/LuxoftSDL/sdl_core/pull/70/commits/6b8a138a35e6abea6bf9b6841a0d21d96c2bc431